### ISSUE TITLE
API: Provide Counts of Photos and Albums 

### ISSUE THREAD

**Issue by thielepaul** [photoprism/photoprism#244]
The API should provide the count of photos and albums:
* [ ] total number of photos (available in `clientConfig.count.Photos`, but not exposed via the API)
* [x] number of photos per month (provided by `/moments/time` endpoint)
* [x] number of photos per album (provided by `/albums` endpoint
* [ ] number of archived photos
* [ ] number of albums (available in `clientConfig.count.Albums`, but not exposed via the API)

Photo counts should be consistent with the number of photos found via `/photos`:
* [ ] number of photos in albums should not include archived photos
* [ ] fix: if a duplicate is placed in the `Originals` folder and then indexed it will be included in the counts but not found via the `/photos` endpoint (similar behavior occurs for related photos)
* [ ] fix: if a photo is deleted from disk, it is still included in the count but not found via the `/photos` endpoint

**Comment by graciousgrey** [comment]
What's the current status? Do you need help from us?

**Comment by thielepaul** [comment]
The current status, as far as I know it, is as indicated by the checkmarks. I do not expect that I will work on PhotoPrism project in the near future, as I recently started a new full-time job and my spare time is limited.

If you think this issue is important, feel free to work on it, I currently do not plan to do so.

**Comment by lastzero** [comment]
@thielepaul That's sad to hear, but of course your job has priority. Will the mobile app continue to be developed or shall we find new maintainers?

We were actually planning to get in contact with you (both) in preparation of our stable release. That's also why we just went through all open issues.

**Comment by thielepaul** [comment]
Yes, that does also include the mobile app, so it would be good to find new maintainers for it.

I still believe that there is a need for a self-hosted open source alternative to Google Photos, so thank you for working on PhotoPrism!
I am also looking forward to check out the stable release of PhotoPrism :)

**Comment by lastzero** [comment]
You're always welcome to contribute when you have time. Thanks for all the work you put into this and sorry for not reaching out earlier!
